tools/interactive/interactivetool_geoexplorer.xml

```diff
@@ -0,0 +1,69 @@
+<tool id="interactive_tool_geoexplorer" tool_type="interactive" name="geoexplorer" version="0.1">
+    <description>An interactive spatial analysis platform using ggvis and Leaflet</description>
+    <requirements>
+        <container type="docker">ylebras/geoexplorer-docker</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="geoexplorer visualisation" requires_domain="True">
+            <port>3838</port>
+            <url>/sample-apps/SIG/</url>
+        </entry_point>
+    </entry_points>
+    <environment_variables>
+
+        <environment_variable name="HISTORY_ID" strip="True">${__app__.security.encode_id($outfile.history_id)}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
+        <environment_variable name="REMOTE_HOST">${__app__.config.galaxy_infrastructure_url}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
+        <environment_variable name="GALAXY_WEB_PORT">8080</environment_variable>
+        <environment_variable name="GALAXY_URL">${__app__.config.galaxy_infrastructure_url}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
+        <environment_variable name="API_KEY" strip="True">
+            #if $__user__:
+                #for $api_key in $__user__.api_keys:
+                    ${api_key.key}
+                    #break
+                #end for
+            #end if
+        </environment_variable> <!-- FIXME: We should have a better way to get user's API key -->
+    </environment_variables>
+    <command><![CDATA[
+
+        /usr/bin/shiny-server.sh
+
+    ]]>
+    </command>
+    <inputs>
+        <param name="infile" type="data" format="csv" label="csv file"/>
+    </inputs>
+    <outputs>
+        <!--data name="outfile" format="txt" /-->
```

Comment:
for now you want to have a random output ... this indicates that the IT is running.

Reply:
arf, in fact no output for now, "just" visualization of data... But we need to configure possible Galaxy history uploads...

Reply:
We need to add an output even if for now the IT is not outputting anything to Galaxy history ?